### DISCUSSION TITLE
Airport CEO icon

### DISCUSSION THREAD

**Discussion by dferdo61** [elrumo/macOS_Big_Sur_icons_replacements#1278] [Icon request] [answered]
https://www.airportceo.com/

Original macOS icon (for reference):
![image](https://user-images.githubusercontent.com/22680916/130153151-13edf205-e2da-4ab3-b820-5a74c1b4c3ed.png)

**Comment by OMGzilla-git** [comment, marked as answer]
Done. You can pull it from my profile while waiting for it to be approved.
https://macosicons.com/u/OMGzilla

**Reply by OMGzilla-git** [reply]
I had to re-create and re-submit the icon. For some reason, it never got approved and went missing while still technically in the queue to be approved.
